apps/speed.c

```diff
@@ -600,6 +600,7 @@ typedef struct loopargs_st {
     unsigned char *buf2_malloc;
     unsigned char *key;
     unsigned int siglen;
+    size_t sigsize;
```

Comment:
Why do you keep siglen?  Should we be worried that we now have *two* fields expressing the same thing?

Reply:
Because its used lots of other places and needs be an int in those locations.

Reply:
Ok...  as long as they aren't used interchangeably, it's fine with me